tests/pytests/conftest.py

```diff
@@ -38,6 +38,16 @@ def sdb_etcd_port():
     return get_unused_localhost_port()
 
 
+@pytest.fixture(scope="session")
+def rabbitmq_port():
+    return "5672"
+
+
+@pytest.fixture(scope="session")
+def rabbitmq_management_port():
+    return "15672"
+
+
```

Comment:
**suggestion** for tests it's a good idea to get an unused port - it may not be necessary here, but if multiple tests are running on the same machine (or the machine already has a different rabbitmq running) this can cause problems.